source/quickstart.rst

```diff
@@ -14,16 +14,15 @@ Here is an example:
 .. code:: javascript
 
     let stack: Stack = new Stack;
+    asdklj asdasd = asdas;
```

Comment:
Can you remove this please @mayanksharma3

Reply:
oops

Reply:
Was doing that to prove that I don't get any warnings for "bad js"